File: Projeto-Greate-People-Enterprise/Back-end/Projeto_RH/Cadastro_Empresas/Model/Empresa.cs
using System.ComponentModel.DataAnnotations;

namespace Cadastro_Empresas.Model
{
    public class Empresa
    {
        public int Id { get; set; }
        public string? Nome { get; set; }
        public string? RazaoSocial { get; set; }
        public string? Cnpj { get; set; }
        public int IdEndereco { get; set; }
        public Endereco EnderecoInfo { get; set; }
        public string? Telefone { get; set; }
        public string? Site { get; set; }
    }
}

File: Projeto-Great-People-Company/Front-end/app-empresa/src/app/interface/EmpresaApi/empresa.ts
import { Endereco } from "./endereco";

export interface Empresa {
    id?: number;
    nome: string;
    razaoSocial: string;
    cnpj: string;
    idEndereco?: number;
    enderecoInfo?: Endereco;
    telefone: string;
    site: string;
}